python/shotgun_model/shotgun_hierarchy_model.py

```diff
@@ -255,6 +267,30 @@ def canFetchMore(self, index):
     ############################################################################
     # protected methods
 
+    def _check_hierarchy_is_supported(self):
+        """
+        Checks the current Shotgun connection to make sure it supports
+        hierarchy queries.
+        """
+
+        # get the current shotgun connection
+        current_engine = sgtk.platform.current_engine()
+        sg_connection = current_engine.shotgun
+        server_caps = sg_connection.server_caps
```

Comment:
Uh oh, I've used `server_caps` a few times in different places. :| 

Yeah, we should definitely discuss...

```python
>>> shotgun.server_caps.version
(7, 0, 0)
>>> shotgun.nav_expand("/", "Version.entity")
Traceback (most recent call last):
  File "<console>", line 1, in <module>
AttributeError: 'ShotgunWrapper' object has no attribute 'nav_expand'
```

Added to hot topics for discussion.